pipeline/projects/sales/objects.py

```diff
@@ -139,8 +139,12 @@ def _populate_object_catalog_record(self, data:dict, parent, lot, cno, rec_num,
 		creation.carried_out_by = self.helper.static_instances.get_instance('Group', 'gpi')
 		row.created_by = creation
 		row.identified_by = self.helper.gpi_number_id(rec_num, vocab.StarNumber)
-
-		record = vocab.EntryTextForm(ident=record_uri, label=f'Sale recorded in catalog: {lot_object_id} (record number {rec_num})')
+		
+		sale_type = non_auctions.get(cno, data.get('non_auction_flag'))
```

Comment:
The `non_auction_flag` only exists in the sales descriptions (events) data, so this `data.get` call will always return `None`.

Reply:
After testing that with an other type, "Private Contract Sale", I got the proper type for that record, and did not get None